dotnet/Simple.Sauce/SauceOptions.cs

```diff
@@ -13,18 +13,55 @@ public class SauceOptions
     {
         private const string DEFAULT_BROWSER_VERSION = "latest";
         private const string DEFAULT_PLATFORM = "Windows 10";
+        private Browser _browserName = Browser.Chrome;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_browserName' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/SauceLabs/simple_sauce/pullRequest?prid=4945851)